addons/api/addon/serializers/auth-method.js

```diff
@@ -51,6 +57,31 @@ export default class AuthMethodSerializer extends ApplicationSerializer {
     return serialized;
   }
 
+  /**
+   * If `adapterOptions.state` is set, the serialization should
+   * include **only state** and version.  Normally, this is not serialized.
+   * @param {Snapshot} snapshot
+   * @return {object}
+   */
+  serializeLDAP(snapshot) {
+    let serialized = super.serialize(...arguments);
+    // Deleting these attributes is a temporary fix.
```

Comment:
Changes in this file are temporary. The reason being that sending these fields to the backend prevents a user from being able to edit the form. There is also a bug related to this, visit this [slack thread](https://hashicorp.slack.com/archives/C014AHXUL3G/p1690405498990289) for more info.

Reply:
I'm ok with a temp fix for now so it does not break the UI, could we add a ticket to track this? The ticket can be linked to the BE ticket so we can clean this up once the fix from BE is in.